attestation-agent/attester/Cargo.toml

```diff
@@ -26,6 +26,7 @@ serde.workspace = true
 serde_json.workspace = true
 serde_with.workspace = true
 sev = { version = "6.2.1", default-features = false, features = ["snp"], optional = true }
+nvml-wrapper = { version = "0.11.0", optional = true, default-features = false, features = ["serde"]}
```

Comment:
Let's move the dep to follow alphabet order for better maintaince

Reply:
fixed